packages/react-native-bridge/ios/GutenbergBridgeDelegate.swift

```diff
@@ -233,6 +233,9 @@ public protocol GutenbergBridgeDelegate: class {
     func gutenbergDidRequestMediaFilesUploadCancelDialog(_ mediaFiles: [[String: Any]])
 
     func gutenbergDidRequestMediaFilesSaveCancelDialog(_ mediaFiles: [[String: Any]])
+    
+    // Crash logging - Sentry
+    func gutenbergDidLogSentryEnvelope(_ envelope: [String: Any])
```

Comment:
The React Native SDK sends the events as [envelopes](https://develop.sentry.dev/sdk/envelopes/), it's basically an object that contains all the data required to ingest an event into Sentry. This function will be used by the Sentry native module to send the events to the main apps ([code reference](https://github.com/Automattic/Automattic-Tracks-iOS/pull/183/files#diff-697ae4f3272c98d9b6ade341612cec5bfb4188e93488ba52d12bd767f8745882R337)).